pages/admin/request/[requestId].tsx

```diff
@@ -150,6 +150,7 @@ export default function Request() {
       </GridItem>
       <GridItem rowSpan={12} colSpan={5} marginTop={5} textAlign="left">
         <PersonalInformationCard
+          // TODO: Temporary fix - will be addressed in API hookup
```

Comment:
what's the fix here, don't see any changes in this file?

Reply:
Good catch, will remove this!